Jenkinsfile_k8s

```diff
@@ -1,18 +1,169 @@
-#!/usr/bin/env groovy
+def recordDeployment(owner, repo, status, environmentURL, environment = "preview", description = "Deploy to preview environment") {
+  withCredentials([usernamePassword(credentialsId: 'github-app-infra', usernameVariable: 'GITHUB_APP', passwordVariable: 'GH_TOKEN')]) {
+    def json = writeJSON(returnText: true, json: [
+        "ref": env.GIT_COMMIT,
+        "environment": environment,
+        "description": description,
+        "required_contexts":[],
+        "auto_merge":false,
+        "transient_environment":false,
```

Comment:
shouldn't this be `true`?

Reply:
Damian says those urls never get deleted, so I marked it as false.

Reply:
if you don't do this any other PR deploying will set the deployment to inactive

https://docs.github.com/en/rest/reference/deployments#inactive-deployments

Reply:
Hrm. So i went with disabling that behavior, because its not transient URL. I don't know what the right behavior is though. This should prevent that issue though.